### PR TITLE
Allow alternate install path

### PR DESCRIPTION
**Pull request by LMWF**
# Description

Add an `--install-path` arg to allow users to install Dapr to a different dir.

Also, add more detailed message if the install to /usr/local/bin (the default) fails because of permissions.

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: https://github.com/dapr/cli/issues/79, https://github.com/dapr/cli/issues/168

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: pkg/standalone/standalone.go [hunk 1]
```diff
@@ -134,7 +137,7 @@ func getDaprDir() (string, error) {
 	return p, nil
 }
 
-func runRedis(wg *sync.WaitGroup, errorChan chan<- error, dir, version string, dockerNetwork string) {
+func runRedis(wg *sync.WaitGroup, errorChan chan<- error, dir, version string, dockerNetwork string, installLocation string) {
```
Comment (amanbha) [inline pkg/standalone/standalone.go]: This function doesnt amek use of installLocation, so param can be removed.

File: pkg/standalone/standalone.go [hunk 2]
```diff
@@ -191,7 +194,7 @@ func isContainerRunError(err error) bool {
 	return false
 }
 
-func runPlacementService(wg *sync.WaitGroup, errorChan chan<- error, dir, version string, dockerNetwork string) {
+func runPlacementService(wg *sync.WaitGroup, errorChan chan<- error, dir, version string, dockerNetwork string, installLocation string) {
```
Comment (amanbha) [inline pkg/standalone/standalone.go]: This function doesnt amek use of installLocation, so param can be removed.

File: pkg/standalone/standalone.go (hunk 1 above)
Comment (LMWF) [reply pkg/standalone/standalone.go]: Unfortunately this is needed to fit the signature of the `initSteps` func.

File: pkg/standalone/standalone.go (hunk 2 above)
Comment (LMWF) [reply pkg/standalone/standalone.go]: Unfortunately this is needed to fit the signature of the `initSteps` func.

File: pkg/standalone/standalone.go (hunk 1 above)
Comment (youngbupark) [reply pkg/standalone/standalone.go]: nit: It would be good to leave a comment for this.

Comment (LMWF) [reply pkg/standalone/standalone.go]: Added

**Pull request merged**